package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "chursj-julienberube",
+  "version": "1.0.0",
+  "private": true,
+  "homepage": "https://jberube.github.io/chursj/",
+  "repository": "git@github.com:jberube/Frontend-Coding-Challenge.git",
+  "dependencies": {
+    "axios": "^0.18.0",
+    "node-noop": "^1.0.0",
```

Comment:
I usually use it in production code to avoid passing different instances of a non-operative functions, nor having to check `if (typeof someFn === 'function')`.

If needed, I can always use `if (someFn === noop)`, but usually I just invoke the function, unless it is in a huge loop. Function invocation in itself is pretty cheap and the `node-noop` function probably gets optimized away by the VM in the end.

This is what is done in angular 1.x source code.